src/ngraph/op/convolution.cpp

```diff
@@ -204,30 +204,35 @@ op::v1::ConvolutionBackpropData::ConvolutionBackpropData(const Output<Node>& dat
     constructor_validate_and_infer_types();
 }
 
+bool op::v1::ConvolutionBackpropData::is_dynamic() const
+{
+    bool is_dynamic = Node::is_dynamic();
+    if (get_inputs().size() == 3 && !is_dynamic)
+    {
+        return !is_type<op::Constant>(input_value(2).get_node());
```

Comment:
We don't have to check for batch and channel dims since they're already checked here: https://github.com/NervanaSystems/ngraph/blob/aosewski/group_conv_bprop_v1/src/ngraph/node.cpp#L954-L964 while data and filters inputs are checked. The third input `output_shape` must be additionally checked, since it's not sufficient to know whether it's shape is dynamic. At this moment there's no support for _dynamic_ output shape.

Reply:
Agree. Missed this line.